boards/riscv/tl321x/Kconfig.board

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2023 Telink Semiconductor
```

Comment:
In all new files use 2024
In modified file use XXXX-2024

Reply:
Okay, will update year with related in the following commits.